### PR TITLE
DT-831 - add i18n to schedules pages

### PR DESCRIPTION
**Pull request by rossedfort**
## Description & motivation 💭 
- Add internationalization to schedules index, view, create, and edit pages.
### Screenshots (if applicable) 📸 
- n/a
### Design Considerations 🎨 
- n/a
### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### CODE REVIEW

File: src/lib/components/schedule/schedule-form-view.svelte [hunk 1]
```diff
@@ -31,13 +33,18 @@
   let namespace = $page.params.namespace;
   let scheduleId = $page.params.schedule;
 
-  let title = schedule ? 'Edit Schedule' : 'Create Schedule';
-  let loadingText = schedule ? 'Editing Schedule...' : 'Creating Schedule...';
-  let backTitle = schedule ? 'Back to Schedule' : 'Back to Schedules';
+  let title = translate('schedules', schedule ? 'edit' : 'create');
+  let loadingText = translate('schedules', schedule ? 'editing' : 'creating');
```
Comment (Alex-Tideman) [inline src/lib/components/schedule/schedule-form-view.svelte:37]: 'edit' and 'create' seem like a common.json term, not specific to schedules

File: src/lib/components/schedule/schedule-form-view.svelte [hunk 2]
```diff
@@ -31,13 +33,18 @@
...
+    'schedules',
+    schedule ? 'back-to-schedule' : 'back-to-schedules',
+  );
   let backHref = schedule
     ? routeForSchedule({ namespace, scheduleId })
     : routeForSchedules({ namespace });
-  let confirmText = schedule ? 'Save' : 'Create Schedule';
+  let confirmText = schedule
+    ? translate('save')
+    : translate('schedules', 'create');
```
Comment (Alex-Tideman) [inline src/lib/components/schedule/schedule-form-view.svelte:47]: Same here

File: src/lib/components/schedule/schedule-form-view.svelte (hunk 1 above)
Comment (Alex-Tideman) [reply src/lib/components/schedule/schedule-form-view.svelte:37]: nvm it's Creating Schedules...

Comment (rossedfort) [reply src/lib/components/schedule/schedule-form-view.svelte:37]: The copy here is "Edit Schedule", and "Create Schedule", so we can make them common if we remove "Schedule", just didn't want to change existing copy.

**Review by Alex-Tideman — APPROVED** [review]

File: src/lib/components/schedule/schedule-form-view.svelte (hunk 2 above)
Comment (rossedfort) [reply src/lib/components/schedule/schedule-form-view.svelte:47]: Same response as above, the copy is "Create Schedule". If it were just "Create", def would have put it in common.

File: src/lib/components/schedule/schedule-form-view.svelte (hunk 1 above)
Comment (Alex-Tideman) [reply src/lib/components/schedule/schedule-form-view.svelte:37]: Yup realized that looking at the json, it's totally fine. We can change next time there is a 'Create text'.

**Pull request merged**